### ISSUE TITLE
Expose statistics and cluster API's

### ISSUE THREAD

**Issue by bakins** [coreos/go-etcd#191]
Any plans to expose endpoints such as `/v2/stats/*` and `/v2/members` in this client?

**Comment by yichengq** [comment]
We don't have that plan so far.
The client pkg https://github.com/coreos/etcd/tree/master/client has
implemented /v2/members. You could give it a try.

**Comment by bakins** [comment]
Thanks. I will take a look at that. It's somewhat confusing that there are two clients, however.

**Comment by yichengq** [comment]
It is in experimental stage, and we are pushing forward step by step.

**Comment by victorcoder** [comment]
Is this https://github.com/coreos/etcd/tree/master/client going to replace go-etcd at some point?

**Comment by philips** [comment]
@victorcoder Yes, the intention is to replace go-etcd with the github.com/coreos/etcd/client package.
